src/main/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/casenotes/CaseNotesByPrisonerMigrationService.kt

```diff
@@ -84,10 +84,13 @@ class CaseNotesByPrisonerMigrationService(
             isMergeDuplicate(it, mergeCopiedCaseNote)
           }
           if (originals.isEmpty()) {
-            log.warn("No duplicates found for offenderNo: $offenderNo, merged case note ${mergeCopiedCaseNote.caseNoteId}")
+            log.warn("No original found for offenderNo: $offenderNo, merged case note ${mergeCopiedCaseNote.caseNoteId}")
           } else {
             copiedNomisIdSet.add(mergeCopiedCaseNote.caseNoteId)
           }
+          if (originals.size > 1) {
+            log.warn("More than 1 originals found for offenderNo: $offenderNo, merged case note ${mergeCopiedCaseNote.caseNoteId}")
+          }
```

Comment:
migration code tweak - I dont think this should happen so would like to know if it does